modules/tinymce/src/core/main/ts/init/Init.ts

```diff
@@ -59,7 +59,7 @@ const trimLegacyPrefix = function (name: string) {
 const initPlugins = function (editor: Editor) {
   const initializedPlugins = [];
 
-  Tools.each(editor.settings.plugins.split(/[ ,]/), function (name) {
+  Tools.each(editor.getParam('plugins').split(/[ ,]/), function (name) {
```

Comment:
I didn't move it because you earlier said that plugins shouldn't be moved, so I left this one where it was. Did I misunderstand it?

Reply:
As I said above, that's because it wasn't a setting available to adjust the plugin behaviour. This however is core, so it is an actual setting that applies to the core.

Reply:
So to be clear, what you've got in the plugins is good. Core, however, should have this in `Settings.ts`.

Reply:
Gotcha.